providers/linux/host_linux.go

```diff
@@ -210,6 +215,24 @@ func (r *reader) hostname(h *host) {
 	h.info.Hostname = v
 }
 
+func (r *reader) fqdn(h *host) {
+	const cmdPath, args = "/bin/hostname", "-f"
```

Comment:
Since you asked about this this morning, I assume you know, but beat's security policy usually prevents us from exec'ing out like this.

Reply:
Oh, I didn't know, thanks for the heads up